admin-dev/themes/new-theme/js/pages/category/index.js

```diff
@@ -75,9 +72,10 @@ $(() => {
   const showcaseCard = new ShowcaseCard('categoriesShowcaseCard');
   showcaseCard.addExtension(new ShowcaseCardCloseExtension());
 
-  new TranslatableField();
-  new TinyMCEEditor();
-  const translatorInput = new TranslatableInput();
+  window.prestashop.components.translatableField.enable();
+  window.prestashop.components.tinyMCEEditor.enable();
+  window.prestashop.components.translatableInput.enable();
```

Comment:
This double call seems redundant, it would be nice to enable an get the component in one call:
```
const translatorInput = window.prestashop.components.translatableInput.enable();
```

But the function name would not we adapted any more 🤔  maybe we should use `getInstance` instead like in a singleton

Reply:
I like the idea to change it to getInstance. The only thing I am not sure about is what if I don't need the instance, I only need to init it. just leave window.prestashop.components.translatableInput.getInstance() without assigning  anything to it? 

It should work but I wonder if this is considered a clean implementation.